opts/charts.go

```diff
@@ -485,3 +487,33 @@ type GraphCategory struct {
 	// The label style of node in this category.
 	Label *Label `json:"label,omitempty"`
 }
+
+// Scatter3DData
+// https://echarts.apache.org/en/option-gl.html#series-scatter3D.data
+type Scatter3DData struct {
+	// name of date item
+	// when name is "", itemStyle is useless
+	Name string `json:"name"`
+
+	// value of date item
+	Value [3]int `json:"value"`
+
+	// user-defined special itemStyle that only useful for this data item
+	ItemStyle *ItemStyle `json:"itemStyle,omitempty"`
+
+	// whether use visualMap style
+	VisualMap bool `json:"visualMap"`
+}
+
+func (s Scatter3DData) MarshalJSON() ([]byte, error) {
```

Comment:
why should we overwrite the MarshalJSON function here?

Reply:
emmm, after testing, it's no need to implement the interface